app/src/test/java/org/kiwix/kiwixmobile/zim_manager/ZimManageViewModelTest.kt

```diff
@@ -517,4 +517,18 @@ class ZimManageViewModelTest {
         .assertValues(StartMultiSelection(viewModel.fileSelectActions))
     }
   }
+
+  @Test
+  fun `finish actionMode on tab change to online section`() {
+    viewModel.libraryTabIsVisible.test()
+      .also { viewModel.currentPage.offer(1) }
+      .assertValue(1)
+  }
+
+  @Test
+  fun `no change in actionMode`() {
+    viewModel.libraryTabIsVisible.test()
+      .also { viewModel.currentPage.offer(0) }
+      .assertEmpty()
```

Comment:
Here, since the filter condition won't pass, no value will be there and hence I checked that it should be empty which seems to work fine. Please let me know if this should be changed.

Reply:
Test is exactly what I wanted to see but the test name needs a little work as mentioned above